core/app/models/spree/pricers/conservative_line_item_pricer.rb

```diff
@@ -0,0 +1,23 @@
+module Spree
+  module Pricers
+    class ConservativeLineItemPricer
```

Comment:
I don't think the Conservative portion of the name adds anything. This is just a standard line item pricer, so I'd call it `LineItemPricer` or `StandardLineItemPricer`.